spec/requests/saml_sessions_spec.rb

```diff
@@ -29,28 +29,11 @@
       expect(session["feedback_return_path"]).to eq destroy_provider_session_path
     end
 
-    context "no mock saml" do
-      before do
-        allow(Rails.configuration.x.laa_portal).to receive(:mock_saml).and_return("false")
-      end
-
-      it "redirects to the SAML sign_out URL" do
-        subject
-        expect(response).to redirect_to(new_feedback_path)
-      end
-    end
-
-    context "mock_saml" do
-      before do
-        allow(Rails.configuration.x.laa_portal).to receive(:mock_saml).and_return("true")
-      end
-
-      after { allow(Rails.configuration.x.laa_portal).to receive(:mock_saml).and_return("false") }
-
-      it "redirects to providers root" do
-        subject
-        expect(response).to redirect_to(providers_root_url)
-      end
+    it "redirects to the feedback page" do
+      subject
+      expect(response).to redirect_to(new_feedback_path)
+      follow_redirect!
+      expect(page).to have_content(I18n.t("saml_sessions.destroy.success"))
```

Comment:
i'd decouple the test from the translation

Reply:
I don't really care either way. 

I think there are benefits to using translations and benefits to not using them.

In this instance, I referred to the translation since that is what the existing test did already —but I can change it if we prefer.